RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -92,10 +88,16 @@ struct WrongPlatformView: View {
                     }
                 }
             }
-            if let url = supportURL {
+
+            if let url = supportInformation?.supportURL(localization: localization),
+               URLUtilities.canOpenURL(url) || RuntimeUtils.isSimulator {
```

Comment:
Does `URLUtilities.canOpenURL(url)` return false in the simulator?

Maybe this should just be

```
if let url = supportInformation?.supportURL(localization: localization),
               URLUtilities.canOpenURL(url) {
```

Reply:
I think some URL schemes that would work on a real device might return `false` in the iOS Simulator (e.g. a `tel://` URL). So with this check we make sure that for any valid URL, even on simulator, we show the contact support button

Reply:
I see, so if it's not a simulator, it will show if it can open it. If it's a simulator, it will always show. Got it! Thank you!!